www/src/components/marketplace/Marketplace.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react'
+import { Button, Div, Flex } from 'honorable'
+
+import { FiltersIcon } from 'pluralsh-design-system'
+
+import MarketplaceSidebar from './MarketplaceSidebar'
+import MarketplaceRepositories from './MarketplaceRepositories'
+
+function Marketplace({ installed }) {
+  const [areFiltersOpen, setAreFiltersOpen] = useState(false)
+
+  return (
+    <Flex
+      p={2}
+      direction="column"
+      overflow="hidden"
+      flexGrow={1}
+      maxHeight="100%"
+    >
+      <Flex flexShrink={0}>
+        Top
+        <Div flexGrow={1} />
+        <Button
+          tertiary
+          small
+          startIcon={<FiltersIcon />}
+          onClick={() => setAreFiltersOpen(x => !x)}
+        >
+          Filters
+        </Button>
+      </Flex>
+      <Flex
+        id="1"
+        mt={1}
+        flexGrow={1}
+        maxHeight="100%"
+      >
+        <Flex
+          direction="column"
+          flexGrow={1}
+        >
+          <MarketplaceRepositories
+            installed={installed}
+          />
+        </Flex>
+        {areFiltersOpen && (
+          <Div
+            ml={2}
```

Comment:
same here, more non-self-documenting settings that are going to be tricky for new devs

Reply:
Replied in Slack